pages/docs/tutorials/native/mpp-ios-android.md

```diff
@@ -0,0 +1,493 @@
+
+# Using Common Library from Android
+
+There are several ways to integrate Android Gradle project with Kotlin multiplatform projects. 
+It is possible to use the `kotlin-multiplatform` plugin directly in Android Gradle project,
+instead of the `kotlin-android`. For that tutorial, we minimize Android project changes and
+created the `:SharedCode` project, and add the ordinary dependency on that project from the
+Android project. You may want to learn more from the
+multiplatform projects [documentation](#). 
+
+Let's include the dependency from the `SharedCode` project to the Android project. We need to patch
+the `app/build.gradle` file and include the following line into the `dependencies { .. }` block:
+
+<div class="sample" markdown="1" mode="groovy" theme="idea" data-highlight-only="1" auto-indent="false">
+
+```groovy
+    implementation project(':SharedCode')
+```
+</div>
+
+Remember, the idea to use common code to generate the text. For that, we need to
+assign the Id to the `TextView` control of our activity to access it from the code. 
+Let's patch the
+`app/src/main/res/layout/activity_main.xml` file
+(the name may be different, if you changed it in the new project wizard)
+and add several more attributes attributes to the `<TextView>` element: 
+```
+        android:id="@+id/main_text"
+        android:textSize="42sp"
+        android:layout_margin="5sp"
+        android:textAlignment="center"
+```
+
+Next, let's include the following line of code into the `MainActivity` class
+from the `/app/src/main/java/<package>/MainActivity.kt` file, to 
+the end of the `onCreate` method:
+
+```
+findViewById<TextView>(R.id.main_text).text = createApplicationScreenMessage()"
+```
+
+Use the intention from an IDE to include the missing import line:
+```kotlin
+import com.jetbrains.jonnyzzz.common.createApplicationScreenMessage
+```
+into the same file. 
+
+That are all changes we needed. Let's see how it works. 
+
+## Running Android Project in Android Studio
+
+At that point, we shall be able to compile and run the project. Let's click on the `App` run configuration
+to have our project running either on a real Android Device or in the emulator. 
+
+![Start the Application]({{ url_for('tutorial_img', filename='native/mpp-ios-android/studio-start-app.png') }})
+
+And so we see the Application running in the Android emulator:
+    
+![Emulator App]({{ url_for('tutorial_img', filename='native/mpp-ios-android/android-emulator-kotlin-rocks-android.png') }}){: width="30%"}
+
+
+# Creating iOS Application
+
+Let's create an iOS application in Xcode. We open Xcode and select *Create a new Xcode project* option. In 
+the dialog, we pick the iOS target, and select the *Single View App*. Fill the next page with defaults, 
+and use the `KotlinIOS` as the *Product Name*. Let's pick Swift as the language (it is possible to use
+Objective-C too). You should instruct Xcode to place the project into the `native` folder under our project. 
```

Comment:
Why "native"?

Reply:
It is needed for the relative paths later in the Framework setup for iOS